src/main/java/pl/zaprogramujzycie/mma/dto/response/MedicineResponse.java

```diff
@@ -7,7 +7,7 @@ public record MedicineResponse(
         long id,
         String name,
         OffsetDateTime expirationDate,
-        int periodAfterOpening
-
+        int periodAfterOpening,
+        long owner
```

Comment:
Skoro to jest teraz family to moze nazwij to familyId?

Reply:
ok